components/Button/src/stories/bem.stories.mdx

```diff
@@ -29,26 +29,319 @@ import readme from "../../README.md";
   {/* eslint-disable react/no-unknown-property */}
   <Story name="Default">
     <button class="denhaag-button">Button</button>
+    <br />
```

Comment:
Why the `<br />` ? This is shown to the one who tries to read the documentation. This must be as clean as possible.

I would rather suggest to add a storybook.css file you import which includes something like `.denhaag-button + .denhaag-button { margin-block-start: 1rem; }`

Reply:
@paulAcato the <br /> are purely for storybook, not for styling

Reply:
Yea i know that. But the Storybook canvas must only show what it needs to show. 

There's also a button-group that adds spacing between buttons. This solution is quick and ugly. If a new developer (or someone without much HTML knowledge)  opens it and sees 3 to 10 `<br/>` will get quite confused.